ui/packages/catalog-ui-search/src/main/webapp/component/query-custom/query-custom.view.js

```diff
@@ -0,0 +1,100 @@
+/**
```

Comment:
Are any of the files in component/query-custom needed anymore?  With your refactor to reuse QueryAdvanced for custom forms removed all the references to these files